srcpkgs/SPOOLES/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'SPOOLES'
+pkgname=SPOOLES
```

Comment:
This package should be called `spooles`.

Reply:
I too find the uppercase name ugly, but the vendor uses uppercase letters when they speak about their software. The folder name after tgz extraction is also uppercase. So I followed the xbps-src manual recommandation ("As a rule of thumb, if the tar.gz of a package contains uppercase letter, then the package name should contain them too").

I think the reason why the name is uppercase is that the library name is an acronym (for "SParse Object Oriented Linear Equations Solver").

So in my opinion the package name should be kept uppercase. Do you agree?

Reply:
Subtle. Let's go with `spooles` then.